pkg/controller/controller_instance.go

```diff
@@ -764,7 +764,7 @@ func (c *controller) reconcileServiceInstance(instance *v1alpha1.ServiceInstance
 		if !isProvisioning {
 			reason = errorErrorCallingUpdateInstanceReason
 		}
-		s := fmt.Sprintf("Error %v ServiceInstance \"%s/%s\" of ClusterServiceClass %q at ClusterServiceBroker %q: %s", provisioningOrUpdatingText, instance.Namespace, instance.Name, serviceClass.Spec.ExternalName, brokerName, err)
+		s := fmt.Sprintf(`ServiceInstance "%v/%v": Error %v ServiceInstance of ClusterServiceClass %q at ClusterServiceBroker %q: %s`, instance.Namespace, instance.Name, provisioningOrUpdatingText, serviceClass.Spec.ExternalName, brokerName, err)
```

Comment:
I think the message here should just be: `Error communicating with broker for <operation>`